jekyll/_cci2/style/formatting.adoc

```diff
@@ -4,19 +4,34 @@
 :toc: macro
 :toc-title:
 
+[#styling-text]
 == Styling text
 
 * Use **bold** for GUI menu options and button text. For example:
 ** Select **Organization Settings** from the sidebar
 ** Select **VCS** in the **Project Settings** window
 ** Select **Projects** from the sidebar and click **Set Up Project** for the project you want to follow
+* Also use **bold** to highlight key words in intro paragraphs. This helps with SEO.
 
 * Use italics for technical terms, for example:
```

Comment:
I think this is too vague. The majority of our docs are technical terms and we don't use italics for most things. Can we make this much more specific?